pages/profile/edit_email.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { NativeBaseProvider } from "native-base";
+import {StyleSheet, Text, TextInput, View, SafeAreaView, TouchableOpacity} from 'react-native';
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { AntDesign } from '@expo/vector-icons'; 
+
+
+type RootStackParamList = {
+    Edit: undefined;
+    EditEmail: undefined;
+};
+
+type editEmailScreenNavigationProp = NativeStackNavigationProp<
+    RootStackParamList,
+    "EditEmail"
+>;
+
+type Props = {
+    navigation: editEmailScreenNavigationProp;
+};
+
+const EditEmail = ({ navigation }: Props) => {
+
+    const [email, setEmail]= React.useState("");
+    
+    return (
+    <NativeBaseProvider>
+        <SafeAreaView style={styles.container}>
+            <View style={styles.header}>
+                <TouchableOpacity
+                    onPress={()=>navigation.navigate('Edit')}>
+                    <AntDesign name="leftcircle" size={40} color='#88838A' />
+                </TouchableOpacity>
+
+                <Text style={styles.header_text}>Email</Text>
+
+                <TouchableOpacity
+                    onPress={()=>navigation.navigate('Edit')}
+                    style={styles.done_button}>
+                    <Text style={styles.header_text}>Done</Text>
+                </TouchableOpacity>
+            </View>
+            <TextInput 
+                style={styles.info_box}
+                placeholder='Email'
+                value={email}
+                onChangeText={newEmail => setEmail(newEmail)}
+            />
+        </SafeAreaView>
+    </NativeBaseProvider>
+);
+};
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#F8FAF0',
+        alignItems: "center",
+    },
+
+    header: {
+        width:"80%", 
+        flexDirection: 'row', 
+        justifyContent:"space-between", 
+        alignItems:'center', 
+        position:'absolute', 
+        top:'15%', 
+        left:'10%'
+    },
+
+    header_text: {
```

Comment:
In JS and TS, the convention is to use camelCase names instead of underscores so let's switch these names to follow that